.circleci/gitChangedLibs.rb

```diff
@@ -19,9 +19,10 @@
 
 for prfile in prfiles
   path = prfile["filename"]
-  for lib in libsTopoSorted
-    if path.include? lib
-      libsModified.add(lib)
+  libsTopoSorted.each do |lib|
```

Comment:
@brandonpage what do you think?
We would run tests in dependent libraries too.

Reply:
My ruby is a little rusty but conceptually I like it.  👍